kount/client.py

```diff
@@ -79,7 +79,12 @@ def process(self, params):
                 logger.debug(error)
                 raise ValueError(error)
         else:
-            logger.debug("process json: %s", req_json)
+            roundtrip = request.elapsed.total_seconds()
+            merc = headers_api.get("X-Kount-Merc-Id", None)
```

Comment:
`MERC` is a mandatory field and it should get validated during validation. No need to check for it afterwards.